nanoFramework.Json.Benchmark/SerializationBenchmarks/ReferenceTypesSerializationBenchmark.cs

```diff
@@ -120,5 +124,50 @@ public void ArrayList()
                 JsonConvert.SerializeObject(arrayList);
             });
         }
+
+        [Benchmark]
+        public void ClassWithAttributeIgnoreEnabled()
+        {
+            RunInIteration(() =>
+            {
+                // Turn OFF the UseIgnore setting
+                Configuration.Settings.UseIgnoreAttribute = true;
+                JsonConvert.SerializeObject(ignoreTest);
+                // Turn OFF the UseIgnore setting
+                Configuration.Settings.UseIgnoreAttribute = false;
+            });
+        }
+        [Benchmark]
```

Comment:
please add an extra line between the functions, so one is missing before the attribute. Same in the next few functions. It helps for readibility.

Reply:
Sorry for more style errors. Even though I like organized code, I'm not used to checking that aspect quite so much. Plus it was a little messy from moving stuff around to get tests working. Will be better next time, don't want to waste your valuable time.

Reply:
Nothing to apologize about! You're doing great! 👍🏻

Reply:
+1 on José remark, you're doing great! No worry, you should have seen my very first PR on a .NET repository :-D Now, with habit, it's easy to spot them. We've been added StyleCop linter on the IoT repository as it's where we do have the most contributions. Not *yet* into those classes one. That will slowly come but it's not urgent. And anyway, we won't put it in place in the tests as it doesn't make too much sense.
So all good! You'll be indeed much better next time and at some point like us being able to spot those :-D
It's really to make it easier to read the code and navigate.